lib/ProductOpener/Display.pm

```diff
@@ -9198,7 +9198,19 @@ sub display_product_api($)
 	my %response = ();
 
 	$response{code} = $code;
-	my $product_ref = retrieve_product($product_id);
+	
+	my $product_ref;
+
+	my $rev = param("rev");
+	local $log->context->{rev} = $rev;
+	if (defined $rev) {
+		$log->info("displaying product revision") if $log->is_info();
+		$product_ref = retrieve_product_rev($product_id, $rev);
+		$header .= '<meta name="robots" content="noindex,follow">';
```

Comment:
As this is for the API, which does not have any HTML output, setting `$header` should not do anything. I'd just remove it. 🙂

Reply:
Good catch, I'll remove it. Thanks!